build.gradle

```diff
@@ -8,8 +8,15 @@ allprojects{
     apply plugin: 'nva.publication.api.dynamo-db'
 }
 
-tasks.withType(JavaCompile) {
-    sourceCompatibility = JavaVersion.VERSION_21
+def javaVersion = 21
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(javaVersion)
+    }
+}
```

Comment:
This says "Use java 21" and the wrapper will download the correct version of java so you don't have to…